Source/MQTTnet/Implementations/MqttTcpChannel.Uwp.cs

```diff
@@ -39,7 +39,11 @@ public MqttTcpChannel(StreamSocket socket, X509Certificate2 clientCertificate, I
 
             CreateStreams();
 
+#if NETCOREAPP3_0 || NET5_0
+            IsSecureConnection = socket.Information.ProtectionLevel >= SocketProtectionLevel.Tls13;
```

Comment:
In my opinion we do not need this different check. Also 1.2 is "secure" because on older frameworks 1.3 is not supported so "1.2" becomes the "most secure" version available.

Reply:
1.3 is backward compatible i have already checked it. The connection was being made on 1.3, but server was only using up to 1.2 and termshark showed that the established lin was 1.2.

Reply:
What do you mean by different check? This one? 
```cs
#if NETCOREAPP3_0 || NET5_0
```

Reply:
Ah OK then it is fine. In my opinion it can stay at "=> ...Tls12" instead 1.3 because now every connection (which still uses 1.2) is treated as "unsafe".

Reply:
Hmm i'm sorry but your sentence sentence is contradicting itself to the conditional. How it can stay at "=> ...Tls12 if every 1.2 connection which uses 1.2 is treated as "not secure"? If it is "not secure" than i think it should rather be changed? Or am i missing something?

Reply:
Ahhh ok :] i have read it once again and checked the code. Ok this can be changed to the previous one :] but yes 1.2 is insecure for example by military :]

Reply:
I can also change this after merge.